PWGGA/GammaConv/AliAnalysisTaskSigmaPlToProtonPiZeroAOD.cxx

```diff
@@ -1680,6 +1791,39 @@ Int_t AliAnalysisTaskSigmaPlToProtonPiZeroAOD::IsPhotonFromXi0(AliAODConversionP
 	return -1;
 
 }
+//________________________________________________________________________
+Int_t AliAnalysisTaskSigmaPlToProtonPiZeroAOD::IsProtonFromLambda(AliAODTrack* track, TClonesArray* fAODMCTrackArray, Int_t iCut, Double_t fWeightJetJetMC, Int_t fill)
+{//Check if proton comes frome a sigma+
+	Int_t mcID = track->GetLabel();
+	AliAODMCParticle* mcTrack = static_cast<AliAODMCParticle*>(fAODMCTrackArray->At(TMath::Abs(mcID)));
+	if(!mcTrack) return -1;
+	if (mcTrack->GetMother() < 0) return -1;
+	AliAODMCParticle *TrackMother = static_cast<AliAODMCParticle*>(fAODMCTrackArray->At(mcTrack->GetMother())); //mother MC particle object
+	if(!TrackMother) return -1;
+	Int_t codeTrack = mcTrack->GetPdgCode();
+	Int_t codeMother = TrackMother->GetPdgCode();
+	if( (codeTrack==2212) && (codeMother==3122) ){
+		return (TrackMother->GetLabel());
+	} else if((codeTrack==2212) && (codeMother==2212) ){
+		Int_t codeGrandMother = codeMother;
```

Comment:
Just to be sure. Did you intend to not ask for "abs(2212)" here? I'm asking as you changed it everywhere else now.

Reply:
Because here I'm looking only for lambdas and not for anti lambdas